client/landing/gutenboarding/stores/onboard/actions.ts

```diff
@@ -47,6 +47,11 @@ export const resetOnboardStore = () => ( {
 	type: 'RESET_ONBOARD_STORE' as const,
 } );
 
+export const setSelectedSite = ( selectedSite: string ) => ( {
```

Comment:
In Calypso, we use the site ID rather than the slug. I don't feel too strongly right now about this, but it might make sense to follow that pattern. (One rationale is that a site's slug/URL can change, e.g. if the user starts with a free .wordpress.com subdomain, and later upgrades to a paid domain, whereas the ID stays the same.)

Reply:
That makes a lot of sense to me, good idea. I've updated the reducer to be keyed by id and this selector to expect a number.